tests/acceptance/subscribers-test.js

```diff
@@ -198,6 +198,7 @@ describe('Acceptance: Subscribers', function() {
 
             // click the import subscribers button
             await click('.gh-btn:contains("Import CSV")');
+            expect(find('.gh-btn:contains("Import CSV")').hasClass('gh-btn-hover-green')).to.be.true;
```

Comment:
FYI we don't tend to check for classes in tests unless we're testing for them being added/removed dynamically as the result of some other logic, reason being is that we want the tests to be coupled to logic rather than style.